enterprise/migrations/0168_auto_20230222_1621.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.17 on 2023-02-21 18:35
+
+from django.db import migrations, models
+import uuid
+
+
+def create_license_enrollment_uuid(apps, schema_editor):
+    licensed_enrollments = apps.get_model('enterprise', 'LicensedEnterpriseCourseEnrollment')
+    for licensed_enrollment in licensed_enrollments.objects.all():
+        licensed_enrollment.uuid = uuid.uuid4()
+        licensed_enrollment.save()
+
+
+def create_learner_credit_enrollment_uuid(apps, schema_editor):
+    learner_credit_enrollments = apps.get_model('enterprise', 'learnercreditenterprisecourseenrollment')
```

Comment:
curious: does a lower-cased model name actually work as the second `apps.get_model()` arg?